src/pages/api/departures/reverse.ts

```diff
@@ -0,0 +1,32 @@
+import { errorResultAsJson, tryResult } from '@atb/modules/api-server';
+import { handlerWithDepartureClient } from '@atb/page-modules/departures/server';
+import { ServerText } from '@atb/translations';
+import { constants } from 'http2';
+import { z } from 'zod';
+import { type ReverseApiReturnType } from '@atb/page-modules/departures/client';
+
+export default handlerWithDepartureClient<ReverseApiReturnType>({
+  async GET(req, res, { client, ok }) {
+    const latQuery = z.string().safeParse(req.query.lat);
+    const lonQuery = z.string().safeParse(req.query.lon);
+
+    if (!latQuery.success || !lonQuery.success) {
+      return errorResultAsJson(
+        res,
+        constants.HTTP_STATUS_BAD_REQUEST,
+        ServerText.Endpoints.invalidMethod,
+      );
+    }
+
+    const lat = parseFloat(latQuery.data);
+    const lon = parseFloat(lonQuery.data);
+
+    if (lat === 0 || lon === 0) {
+      return ok(undefined);
```

Comment:
Is this actually OK? Should this rather be bad request? Or isn't this possible? Maybe the parsing above should be directly to float so to avoid this double casing?

```
z.number().safeParse(req.query.lat);
```

or whatever

Reply:
Remnants of when I used useSWR and it kept revalidating on focus with 0, 0 (fallback values). 

Wasn't able to safeParse the query string directly to a number when I tried 🤔

Reply:
Cleaned it up some, decided to remove the check for 0 since we have control over what's sent in and not.